people/sources.json

```diff
@@ -18,5 +18,5 @@
   "jana",
   "kyrylo",
   "zuzana",
-  "matej"
-]
```

Comment:
no ale tohle kdyz smazes, tak to nebude platny JSON. Jestli (spravne) pouzivas IDEA, tak by ti to melo rvat. Koukni.